src/Humanizer/TimeSpanHumanizeExtensions.cs

```diff
@@ -53,6 +53,28 @@ public static string Humanize(this TimeSpan timeSpan, int precision, bool countE
             return ConcatenateTimeSpanParts(timeParts, culture, collectionSeparator);
         }
 
+        public static string ToAge(this TimeSpan timeSpan, CultureInfo culture = null)
+        {
+            culture ??= CultureInfo.CurrentUICulture;
+
+            var ageExpression = timeSpan.Humanize(maxUnit: TimeUnit.Year);
+            if (culture.UsesEnglishLanguage())
+                ageExpression += " old";
```

Comment:
Please change to using resource string instead.

Reply:
That's a good idea. I can add something like the `TimeSpanHumanize_AgeSuffix` resource.

Because most languages do not have such a suffix, I'm thinking of making the default resources' TimeSpanHumanize_AgeSuffix = "", but since the default resources correspond to "en-US" it means I'll need to introduce Resources.en.resx and Resources.en-US.resx to override that resource, right? Is there an easier way?

Reply:
The thing is that it might be not a suffix but something else in some random language. So, I think it would be better to have a formatting string like `”{0} age”`.

Yes you would need to introduce default resource with `"{0}”` and appropriate resource for `en` culture.

Reply:
Ok so I introduced `TimeSpanHumanize_Age` & `TimeSpanHumanize_PreHyphenatedAge` format resources.

- Defined default values (in Resources.resx) as `"{0}"` and `""`, respectively
- Had to add Resources.en.resx and Resources.en-US.resx, to override the defaults
- Adapted nuspec files accordingly
- Simplified default (en-US) tests and moved tests for other English-based cultures to another file

Reply:
After further consideration, I think I went about it the wrong way. 🤦‍♂️ 

Conceptually it made sense to have in `Resources.resx` the most common age formats, i.e.
- the basic age format being "{0}", i.e. the same as what `TimeSpan.Humanize()` returns
- the pre-hyphenated age format being "", because in most languages the hyphenated age makes no sense and it too should just be an empty string, right?

The problem is that `Resources.resx` happens to contain the en-US culture's resources. So with the above changes I had to introduce a `Resources.en-US.resx` file to override the default formats, but that's actually adding confusion and forcing people who just want the en-US resources to not only install Humanizer.Core, but also the eventual en-US NuGet package. Yikes...

I therefore made the following changes (which I haven't pushed yet):
- Moved to `Resources.resx` the en-US age formats
- Added to each of the other resx files the so-called most common age formats _(note that people with knowledge of other cultures may eventually need to revisit those resources, especially the Germanic ones mentioned previously)_

Let me know @hazzik whether I should push those changes.

Reply:
Sorry @louis-z I don't really know what would be the best strategy to deal with resources.

Reply:
So I finally went ahead with the proposed changes, to simplify resource management:
- Made the English age and hyphenated age formats the default (i.e. through `Resources.resx`)
- Overrode those settings for French culture, as an example. For most other cultures, similar changes will be needed (should I submit them as part of this PR?)